src/Microsoft.VisualBasic/src/Microsoft.VisualBasic.vbproj

```diff
@@ -79,4 +94,9 @@
     <Reference Include="System.Linq" />
     <Reference Include="System.Reflection.Extensions" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+  <ItemGroup>
+    <Reference Include="System.Security.Permissions">
+      <HintPath>..\..\..\bin\ref\netstandard\System.Security.Permissions.dll</HintPath>
```

Comment:
><HintPath>..\..\..\bin\ref\netstandard\System.Security.Permissions.dll</HintPath> [](start = 6, length = 81)

Is the `<HintPath>` necessary? And should this reference be moved to the `<ItemGroup>` above?